pyads/ads.py

```diff
@@ -597,6 +597,26 @@ def __init__(self, ams_net_id, ams_net_port, ip_address=None):
         self._open = False
         self._notifications = {}  # type: Dict[int, str]
 
+    @property
+    def ams_netid(self):
+        # type: () -> str
+        return self._adr.netid
+
+    @ams_netid.setter
+    def ams_netid(self, netid):
+        # type: (str) -> None
+        self._adr.netid = netid

```

Comment:
It shouln't be possible to change the own net id or port after a connection is opened. Otherwise this is not consistent. Could you add a check here if a conneciton has been opened?